internal/transactionsubmission/README.md

```diff
@@ -1,10 +1,24 @@
 # Transaction Submission Service
 
-The Transaction Submission Service (TSS) is a component that is responsible for submitting payment transactions to the Stellar Network.
+The Transaction Submission Service (TSS) allows for transaction submission at scale to the Stellar network from a single source/distribution account, which is made possible through the use of [channel accounts](https://developers.stellar.org/docs/encyclopedia/channel-accounts) spawned from said distribution account. In the context of SDP, the TSS is responsible for submitting transactions to the network for each respective payment read from a shared database table that is "queued" by the SDP. It works seamlessly by way of graceful error handling and transaction retries alongside mechanisms such as transaction fee bumps and adjustment of the rate at which new transactions are polled when certain errors are encountered.
 
-The SDP will directly 'queue' transactions (create transactions in the database) and the Transaction Submission Service will read these transactions and submit them to the Stellar Network.
+Use of the Transaction Submission Service requires at least a single of the aforementioned channel accounts to be seeded in storage in advanced. We provide a set of CLI's that can be used to create, delete, and change the number of channel accounts the service manages with ease. To learn how to use these commands, please refer to the [Channel Accounts Management](#channel-accounts-management) section below.
 
-The Transaction Submission Service requires channel accounts to be seeded in storage in advanced. To learn how to fulfill this prerequisite, please refer to the [Channel Accounts Management](#channel-accounts-management) section below.
+## Distribution Account
+Though the transaction submitter and channel account management commands allow the distribution private key to be specified upfront through the `distribution-seed` flag, we recommend users to configure it through the environment variable `DISTRIBUTION_SEED` in a one and done way to mitigate risks of exposing this information. For the distribution seed to be valid, the account associated with it must contain a balance of the asset that is specified by the payment/transaction records.
```

Comment:
💡 

```suggestion
Though the transaction submitter and channel account management commands allow the distribution private key to be specified upfront through the `distribution-seed` flag, we recommend users to configure it through the environment variable `DISTRIBUTION_SEED` in a one and done way to mitigate risks of exposing this information. For the distribution seed to be valid, the account associated with it must exist in the network. 

In terms of balance, the distribution account needs to hold some XLM balance in order to pay for gas fees and also to support the creation of Channel accounts (1 XLM is needed per channel account). The distribution account also must contain a balance of the assets that are specified by the payments/transactions records otherwise, these transactions will fail with an error explaining the reason.
```

Reply:
if we're also reading this from a stellar novice's perspective, it may also be helpful to add a quick guide on how to create a distribution account in testnet - I will add what's in the quick start guide here!

Reply:
Good idea!

Reply:
Than part can be added in another PR too, if you prefer. Up to you